scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1031,7 +1031,7 @@ int match_layer(u32 layer_id, pid_t pid, const char *cgrp_path)
 
 	struct task_struct *p;
 	struct layer *layer = &layers[layer_id];
```

Comment:
does layer_id still needs to be bound checked?

Reply:
It depends on how the code is generated but yeah, let's move that below too.